### DISCUSSION TITLE
How to pull stats from a google cloud tasks queue that has finished?

### DISCUSSION THREAD

**Discussion by ollebergkvist** [googleapis/google-cloud-node#6009] [Announcements]
I'm trying to figure out how to pull the following stats from a Google Cloud tasks queue that has finished executing. I'm running a queue Mon, Wed, Fri every week, and I need to get some useful stats once the queue finishes every time.

- Nr of dispatched tasks
- Nr of successful tasks
- Nr of unsuccessful tasks (all retries failed)
- Timestamp queue execution start
- Timestamp queue finished
- Preferably also retrieving the payload of all failed requests if any

I can only find this doc: https://googleapis.dev/nodejs/tasks/3.0.2/google.cloud.tasks.v2beta2.QueueStats.html

But I can't pull the needed data from there.

Or do you need to pull this info from logs explorer or another service?

Preferably I want to configure this as a webhook meaning once a queue finishes, it triggers an endpoint that generates this data.

**Comment by sofisl** [comment]
SInce this is an API/environment question, I'm going to transfer this to Discussions. However,  I wonder if [this]9https://cloud.google.com/tasks/docs/monitor) is what you're looking for?
